scripts/scil_fix_dsi_studio_trk.py

```diff
@@ -127,11 +127,16 @@ def main():
     # LPS -> RAS convention in voxel space
     sft.to_vox()
     flip_axis = ['x', 'y']
-    sft.streamlines._data -= get_axis_shift_vector(flip_axis)
     sft_fix = StatefulTractogram(sft.streamlines, args.in_dsi_fa,
-                                 Space.VOX)
+                                 Space.VOXMM)
+    sft_fix.to_vox()
+    sft_fix.streamlines._data -= get_axis_shift_vector(flip_axis)
+
     sft_flip = flip_sft(sft_fix, flip_axis)
 
+    sft_flip.to_rasmm()
+    sft_flip.streamlines._data -= [0.5, 0.5, -0.5]
```

Comment:
Will it work everytime ?

Reply:
Good enough and there is a warning at the beginning of the script imao it's good to go.